playbooks/roles/ses-rook/tasks/main.yml

```diff
@@ -0,0 +1,104 @@
+---
+- name: Ensure kubectl is properly configured
+  include_role:
+    name: common-deployer
+    tasks_from: kubectl-setup
```

Comment:
Hm, won't this be called again from deploy-osh/airship-setup-deployer ?

Reply:
Maybe airship-setup-deployer will be executed in a most advanced step?

Reply:
yes it will, but unfortunately is needed in this step as well, so we need to execute it before otherwise there is no deploying ses-in-rook. Its idempotent so it should not matter much.

Reply:
OK, we can reorganize the code if we decide we actually want to deploy with rook for real